devops/jobs/ClusterInstanceMonitoring.groovy

```diff
@@ -0,0 +1,114 @@
+/*
+ 
+ This job expects the following credentials to be defined on the folder
+    tools-edx-jenkins-aws-credentials: file with key/secret in boto config format
+    find-active-instances-${deployment}-role-arn: the role to aws sts assume-role
+
+*/
+
+package devops.jobs
+import static org.edx.jenkins.dsl.Constants.common_wrappers
+import static org.edx.jenkins.dsl.Constants.common_logrotator
+
+class ClusterInstanceMonitoring{
+	public static def job = { dslFactory, extraVars ->
+		assert extraVars.containsKey('DEPLOYMENTS') : "Please define DEPLOYMENTS. It should be a list of strings."
+        assert !(extraVars.get('DEPLOYMENTS') instanceof String) : "Make sure DEPLOYMENTS is a list and not a string"
+		extraVars.get('DEPLOYMENTS').each { deployment ->
+			
+			dslFactory.job(extraVars.get("FOLDER_NAME","Monitoring") + "/cluster-instance-monitoring-${deployment}") {
+				wrappers common_wrappers
+                logRotator common_logrotator
+
+                wrappers {
+                    credentialsBinding {
+                        file('AWS_CONFIG_FILE','tools-edx-jenkins-aws-credentials')
+                        def role = "find-active-instances-${deployment}-role-arn"
```

Comment:
I know we came up with this workaround when we were debugging '' vs "".  Do you still need it?